docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,121 @@
+
+## Collections
+
+Within your collection, you can also access the **Users** and **Data Sources** tabs to further customize your collection.
+
+### Users
+
+Within the **Users** tab, review the users and user groups with access to your collection.
+
+Select **Add** to search for a user or user group to add, of which may be assigned as a viewer, editor, or manager.
+
+### Data sources
+
+Within the **Data Sources** tab, review and add source(s) of your data to create reports and dashboards inside the collection.
+
+## Additional analysis
+
+Now that we’ve created data sets within the **Home** page, let’s shift into the **Analysis** tab.
+
+Inside this tab, you’ll notice **Outlier Analysis** and **Branch Analysis**.
+
+### Outlier analysis
+
+Inside **Outlier Analysis**, we utilize heatmap displays. Click **Select Process**, choose your process, and choose your version.
+
+![heatmap example](./assets/heatmap.png)
+
+Within the example above, we notice increased heat (recognized as red) surrounding our invoice approved gateway. Several instances have taken significantly longer than average, so we may choose to take a closer look at these instances by downloading the instance IDs, or viewing the details for further analysis. Here, you can also find if the outliers have a shared variable.
+
+### Branch analysis
+
+Inside the **Branch Analysis** tab, we can select a ​​process and analyze how particular gateway branches impact the probability of reaching an end event.
+
+Fill in the process field, click on a gateway, and choose your end event. In the example below, we can further analyze the likelihood of an invoice being processed once it reaches the gateway for approval:
+
+![branch analysis example](./assets/analysis.png)
+
+Here, we’ve selected a process flow, gateway, and endpoint for a breakdown of all the instances that went through a particular gateway to a specific endpoint. Hover over the gateway for a breakdown of the process itself.
+
+## Additional resources and next steps
+
+We’ve only touched the surface of Optimize. The component is full of additional ways to analyze your data for effective process improvement. We recommend taking a look at several resources to catch up on Optimize’s latest release, new features, and many usage examples:
+- [Camunda Optimize 3.6.0 Release](https://camunda.com/blog/2021/10/camunda-optimize-360-released/)
+- [The Ultimate Guide to Solving Bottlenecks with Camunda: Part 1](https://camunda.com/blog/2021/10/the-ultimate-guide-to-solving-bottlenecks-with-camunda-part-1/)
+- [Camunda Optimize examples](https://github.com/camunda/camunda-optimize-examples)
+- [Process performance made transparent](https://camunda.com/products/camunda-platform/optimize/reports/)
+
+[//]:# (Eric, how exactly does the screenshot automation process work on your team? Do you have particular images you'd like to see in this guide? Happy to meet 1:1 on this if necessary.)
```

Comment:
@christinaausley @HanselIdes don't forget this comment 😄

Reply:
Pulled in @megglos as he's also taking a look at all of the images we migrated over within our [Cloud-friendly Optimize documentation](https://github.com/camunda-cloud/camunda-cloud-documentation/pull/483) (given many will need to be updated as Cloud-first).

We're planning to meet next week to discuss screenshot automation for the images within this guide, so @HanselIdes please do let me know if you have alternative screenshots you would like to use in this guide by the end of this week.

Reply:
I don't have any screenshots that I had explicitly in mind for this guide, and I'm comfortable opening PRs to make incremental edits in the future.